### PR TITLE
feat: Add tempo rock

### PR DESCRIPTION
**Pull request by mmkay**
- [x] I have signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] I am following the [CONTRIBUTING](https://github.com/canonical/oci-factory/blob/main/CONTRIBUTING.md) guidelines

*Ping the @canonical/rocks team.*

---

## Description
This PR adds the Tempo ROCK to the OCI factory.

### Related issues
#171 

---

*Picture of a cool rock:*
0840e201-0006-454d-9ded-b58c7a3b2d0d

### CODE REVIEW

File: oci/tempo/documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    value: '9411:9411'
+    description: Zipkin format ingestion port.
+debug:
+  text: |
+    ### Debugging
+    
+    To debug the container:
+
+    ```bash
+    docker logs -f tempo-container
```
Comment (cjdcordeiro) [inline oci/tempo/documentation.yaml:47]: This is gonna give you the Pebble stdout. Wouldn't it be better to do `docker exec ... pebble logs -f tempo`?

Comment (mmkay) [reply oci/tempo/documentation.yaml]: You're right - that's better. Updated the file.

File: oci/tempo/documentation.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    value: '9411:9411'
+    description: Zipkin format ingestion port.
+debug:
+  text: |
+    ### Debugging
+    
+    To debug the container:
+
+    ```bash
+    docker exec -it tempo-container /bin/pebble logs -f tempo
```
Comment (cjdcordeiro) [inline oci/tempo/documentation.yaml:47]: Thanks for the update. Why do you need `/bin` though? That should be in $PATH. Just `pebble logs ...` should work, doesn't it?

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: oci/tempo/documentation.yaml (hunk 2 above)
Comment (mmkay) [reply oci/tempo/documentation.yaml]: True, it does and I updated the description. Thanks, good catch!

**Review by cjdcordeiro — APPROVED** [review]
LGTM thanks

**Pull request merged**